FIX: Global mesh region + doc updates (#4854)
ansys/pyaedt: doc/source/User_guide/mesh.rst

  :width: 800
  :alt: Mesh object List

Mesh in Icepak
--------------

> Icepak employs a unique approach to mesh operations, managed through mesh regions, which can be edited directly from the pyaedt object.

Reviewer: ```suggestion
Icepak employs a unique approach to mesh operations, managed through mesh regions, which can be edited directly from the PyAEDT object.
```

> **Global Mesh Region**
> 
> The global mesh region object is accessed using:

Reviewer: ```suggestion
**Global mesh region**

This code accesses the global mesh region object:
```
Sentence case titles and active voice.

.. code:: python

    glob_msh = ipk.mesh.global_mesh_region

> This object allows for the editing of global mesh settings through its `settings` property, which functions like a dictionary.

Reviewer 2: ```suggestion
This object allows for the editing of global mesh settings through its `settings` property, which behaves like a dictionary.
```

.. code:: python

    glob_msh.settings["MeshRegionResolution"] = 3

> Available keys differ depending on whether manual or automatic settings are used. This can be adjusted using the `manual_settings` boolean property of the global mesh region object.

Reviewer: ```suggestion
The keys available depend on whether manual or automatic settings are used. You can use the ``manual_settings`` Boolean property of the global mesh region object to chagne the setting.
```

> To modify the global region dimensions, access the `global_region` property and modify its attributes.

Reviewer: ```suggestion
To modify the global region dimensions, access the '`global_region'` property and modify its attributes:
```
Single backtick results it italic formatting in the doc, versus double backticks format as a code entity.

.. code:: python

    glob_msh.global_region.positive_z_padding_type = "Absolute Offset"
    glob_msh.global_region.positive_z_padding = "5 mm"

> To modify the properties of the Region object (for example the gas inside the region), access the `object` property of the `global_region` object:

Reviewer: ```suggestion
To modify the properties of the ``Region`` object (for example the gas inside the region), access the ``object`` property of the ``global_region`` object:
```

.. code:: python

    glob_reg = glob_msh.global_region
    glob_reg.object.material_name = "Carbon Monoxide"

> This is a pointer to the same object accessible from `ipk.modeler["Region"]`.
> 
> The image below summarizes these three objects:

Reviewer: ```suggestion
This is a pointer to the same object accessible from ``ipk.modeler["Region"]``.

The following image summarizes these three objects:
```
Avoid positional words like above and below in case the reader is looking a PDF or printed pages.

.. image:: ../Resources/icepak_global_mesh_region_objects.png
  :width: 80%

>   :alt: Global Mesh objects and sub-objects

Reviewer: ```suggestion
  :alt: Global mesh objects and subobjects
```

> A complete example using the global mesh region is provided:

Reviewer: ```suggestion
This code provides a complete example using the global mesh region:
```

.. code:: python

    ipk = Icepak()
    glob_msh = ipk.mesh.global_mesh_region
    glob_msh.manual_settings = True
    glob_msh.settings["MaxElementSizeX"] = "2mm"
    glob_msh.settings["MaxElementSizeY"] = "3mm"
    glob_msh.settings["MaxElementSizeZ"] = "4mm"
    glob_msh.settings["MaxSizeRatio"] = 2
    glob_msh.settings["UserSpecifiedSettings"] = True
    glob_msh.settings["UniformMeshParametersType"] = "XYZ Max Sizes"
    glob_msh.settings["MaxLevels"] = 2
    glob_msh.settings["BufferLayers"] = 1
    glob_msh.update()

> **Local Mesh Regions**
> 
> To create a mesh region, the `assign_mesh_region` function is used:

Reviewer: ```suggestion
**Local mesh regions**

To create a mesh region, use the ``assign_mesh_region()`` function:
```

.. code:: python

    mesh_region = ipk.mesh.assign_mesh_region(name=object_name)

> The settings of the returned object can be modified using the same approach as with the global mesh region object.

Reviewer: ```suggestion
You can modify the settings of the returned object using the same approach as with the global mesh region object.
```

> To access the subregion that defines the local mesh region and modify its dimensions:

Reviewer: ```suggestion
This code accesses the subregion that defines the local mesh region and modifies its dimensions:
```

.. code:: python

    subregion = mesh_region.assignment
    subregion.positive_z_padding_type = "Absolute Offset"
    subregion.positive_z_padding = "5 mm"

> To access the parts included in the subregion:

Reviewer: ```suggestion
This code accesses the parts included in the subregion:
```

.. code:: python

    subregion.parts

> In AEDT 2024 R1, a significant revamp of the mesh region paradigm has been introduced, resulting in limited support for older versions. To use the same functions in older versions, the region box must be defined first and passed as the first argument of `assign_mesh_region`.

Reviewer: ```suggestion
AEDT 2024 R1 introduced a significant revamp of the mesh region paradigm, resulting in limited support for older versions. To use the same functions in older versions, you must define the region box first and pass it as the first argument of the ``assign_mesh_region()`` function.
```

> **Mesh Operations**
> 
> - To assign a mesh level to some objects, use the `assign_mesh_level` method:

Reviewer: ```suggestion
**Mesh operations**

- To assign a mesh level to some objects, use the ``assign_mesh_level()`` method:
```

  .. code:: python

    ipk.mesh.assign_mesh_level(mesh_order={"Box1": 2, "Cylinder1": 4})

> - To assign a mesh file for reuse to some objects, use the `assign_mesh_reuse` method:

Reviewer: ```suggestion
- To assign a mesh file for reuse to some objects, use the ``assign_mesh_reuse()`` method:
```
